### ISSUE TITLE
will spark work well on depthwise conv?

### ISSUE THREAD

**Issue by LinB203** [keyu-tian/SparK#37]
hi, wonderful work! I wonder that will it work well while `in_channel == group` in `nn.Conv2D`?

**Comment by keyu-tian** [comment]
Yeah for sure, just use that!

**Comment by xubin04** [comment]
I want to know if I can effectively transfer this pre-trained approach to video prediction tasks. My data has dimensions of batch_size, time_stamp*channel, height, width. Can I mask different regions for different timestamps and can depth-wise processing handle this masking situation effectively.And thank you for your gerat work, this is a big step in CNN pretain method.
